docs/maps/asset-tracking-tutorial.asciidoc

```diff
@@ -0,0 +1,340 @@
+    codec => "json"
+  }
+}
+
+filter {
+  split {
+    field => "[resultSet][vehicle]"
+  }
+
+  if ![resultSet][vehicle][inCongestion] {
+    mutate {
+      update => {
+        "[resultSet][vehicle][inCongestion]" => "false"
+      }
+    }
+  }
+
+  mutate {
+    add_field => {
+      "bearing" => "%{[resultSet][vehicle][bearing]}"
+      "in_congestion" => "%{[resultSet][vehicle][inCongestion]}"
+      "location" => "%{[resultSet][vehicle][latitude]},%{[resultSet][vehicle][longitude]}"
+      "route" => "%{[resultSet][vehicle][routeNumber]}"
+      "time" => "%{[resultSet][vehicle][time]}"
+      "type" => "%{[resultSet][vehicle][type]}"
+      "vehicle_id" => "%{[resultSet][vehicle][vehicleID]}"
+    }
+    remove_field => [ "resultSet", "@version", "@timestamp" ]
+  }
+
+  mutate {
+    convert => {
+      "bearing" => "float"
+  "in_congestion" => "boolean"
+      "time" => "integer"
+    }
+  }
+}
+
+output {
+  stdout {
+    codec => rubydebug
+  }
+
+  elasticsearch {
+    cloud_auth => "<username:password>"
+    cloud_id => "<cloud_id>"
+    index => "tri_met_tracks"
+    document_id => "%{[vehicle_id]}_%{[time]}"
+  }
+}
+----------------------------------
+
+. Replace `<tri_met_app_id>` with your TriMet application id.
+. Replace `<username:password>` with your Elastic Cloud deployment credentials.
+. Replace `<cloud_id>` with your {ece}/ece-cloud-id.html[elastic cloud id].
+. Open a terminal window, and then navigate to the Logstash folder.
+. In your `logstash` folder, run Logstash with the TriMet pipeline:
++
+[source,bash]
+----------------------------------
+bin/logstash -f config/trimet-pipeline.conf
+----------------------------------
+
+. Wait for Logstash to initialize and confirm data is flowing. You should see messages similar to this:
++
+[role="screenshot"]
+image::maps/images/asset-tracking-tutorial/logstash_output.png[]
+. Leave the terminal window open and Logstash running throughout this tutorial.
+
+[float]
+==== Step 3: Create a {kib} index pattern for the tri_met_tracks {es} index
+
+. In Kibana, open the main menu, and click *Stack Management > Index Patterns*.
+. Click *Create index pattern*.
+. Give the index pattern a name: *tri_met_tracks**.
+. Click *Next step*.
+. Set the *Time field* to *time*.
+. Click *Create index pattern*.
+
+{kib} shows the fields in your index pattern.
+
+[role="screenshot"]
+image::maps/images/asset-tracking-tutorial/index_pattern.png[]
+
+[float]
+==== Step 4: Explore the Portland bus data
+
+. Open the main menu, and click *Discover*.
+. Set the index pattern to *tri_met_tracks**.
+. Open the <<set-time-filter, time filter>>, and set the time range to the last 15 minutes.
+. Expand a document and explore some of the fields that you will use later in this tutorial: `bearing`, `in_congestion`, `location`, and `vehicle_id`.
+
+[role="screenshot"]
+image::maps/images/asset-tracking-tutorial/discover.png[]
+
+[float]
+=== Part 2: Build an operational map
```

Comment:
The tutorial is ok as is, but do you want to add an intro sentence here to set up this section?